homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  config-flow:
+    status: todo
+    comment: |
+      Data descriptions.
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Comment:
Waht is `test_flow_get_no_username` for? What can the user do about this

Reply:
Making sure they have a username in Sensibo api (which I think is a must anyhow) and retry given the api call failed.

Reply:
I mean more like, if it can't happen, why do we have the code paths for it. If it can happen I think raising an error is strange because it doesn't sound like a temporary issue

Reply:
It would in reality be a temporary issue (that particular api call failed for some reason).
That call is only needed for the setup (and migration from earlier versions) so it's not something to worry about.